pkgs/development/libraries/agda/cubical-mini/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  mkDerivation,
+  fetchFromGitHub,
+  ghc,
+  cabal-install,
+}:
+
+mkDerivation rec {
+  pname = "cubical-mini";
+  version = "nightly-20241214";
```

Comment:
Is it possible to pin this to a stable version? e.g. v0.5?

Reply:
Yes, but then it will only build on Linux, not MacOS. See: https://github.com/cmcmA20/cubical-mini/pull/63

Reply:
@steeleduncan So, can I leave it at nightly?